scripts/populate.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# This script is used to populate the stack with cozy instances
+# It requires updating /etc/hosts
+
+if [ $# != 1 ]
+then
+    echo "Wrong number of arguments!"
+    echo "Usage: populate [number of instances to create]"
+    return
+fi
+
+repository=$(pwd)
+cd ${repository}/scripts
+
+echo "Clearing old webhooks data..."
+rm ../data/webhooks.json
+
+for i in `seq 1 ${1}`
+do
+    domain="test${i}.cozy.localhost:8080"
+    echo "Creating instance ${domain}"
+    # Destroy the instance in case it already exists
+    cozy-stack instances destroy ${domain} --force
+    # Create a new instance
+    cozy-stack instances add --apps drive,photos ${domain} --passphrase cozy
+    cozy-stack instances modify ${domain} --onboarding-finished
+    # Generate token first
+    ACH_token=$(cozy-stack instances token-cli ${domain} io.cozy.bank.operations)
+    # Split our dataset in chunks (deterministic)
+    node split.js ../assets/split.json ${i}
```

Comment:
I think the assets directory should be created at some point by the script if it does not exist?

Reply:
the creation is done by the split.js script